posts/markdown-project.md

```diff
@@ -0,0 +1,147 @@
+
+In 2021 the Open Web Docs team, with help from Mozilla, the W3C, and the wider web docs community, converted the authoring format for MDN Web Docs  - all 11,000 pages of it - from HTML to Markdown. In this post we'll talk about why we did it, how we did it, and how it turned out.
+
+## HTML soup
+
+Before 2020 MDN was a Wiki, and contributors edited pages using a web-based WYSIWYG HTML editor. This made it easy to make casual contributions: people could edit text and apply simple formatting, like **bold** or `code`, without having to edit the underlying HTML. But a WYSIWYG editor is not well-suited to more complex edits, and authors often had to edit the underlying HTML source directly. Also, because the underlying source was hidden by default, HTML cruft, like `<span id="486uw3y3">` crept into the source, often from authors pasting HTML from another rich editing environment into the MDN WYSIWYG editor.
```

Comment:
```suggestion
Before 2020, MDN was a Wiki, and contributors edited pages using a web-based WYSIWYG HTML editor. This made it easy to make casual contributions: people could edit text and apply simple formatting, like **bold** or `code`, without having to edit the underlying HTML. But a WYSIWYG editor is not well-suited to more complex edits, and authors often had to edit the underlying HTML source directly. Also, because the underlying source was hidden by default, HTML cruft, like `<span id="486uw3y3">` crept into the source, often from authors pasting HTML from another rich editing environment into the MDN WYSIWYG editor.
```